django/chat/views.py

```diff
@@ -910,3 +917,120 @@ def update_qa_options_from_librarian(request, chat_id, library_id):
             "trigger_library_change": "true" if library != original_library else None,
         },
     )
+
+
+def generate_prompt(task_or_prompt: str):
```

Comment:
This can be moved to utils.py - trying to keep views.py only for functions that handle requests.